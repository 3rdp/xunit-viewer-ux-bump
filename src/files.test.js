import React from 'react'
import Files from './files'
import renderer from 'react-test-renderer'

test('renders files', () => {
  const tree = renderer.create(<Files />).toJSON()
  expect(tree).toMatchSnapshot()
})
