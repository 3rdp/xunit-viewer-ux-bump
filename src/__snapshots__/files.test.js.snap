// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders files 1`] = `
<div
  className="files card is-inactive"
>
  <header
    className="card-header"
  >
    <button
      className="card-header-icon"
      onClick={[Function]}
    >
      <h6
        className="subtitle is-5"
      >
        <span
          className="icon"
        >
          <i
            className="fas fa-code"
          />
        </span>
        <span>
          Files
        </span>
      </h6>
      <span
        className="icon"
      >
        <i
          className="fas fa-angle-down"
        />
      </span>
    </button>
  </header>
  <div
    className="card-content"
  >
    <div
      className="tabs is-boxed"
    >
      <ul>
        <li
          className="is-inactive"
        >
          <a
            href={null}
            onClick={[Function]}
          >
            <span>
              file.xml
            </span>
            <button
              className="delete is-small"
              disabled={true}
              onClick={[Function]}
            />
          </a>
        </li>
        <li
          className="is-active"
        >
          <a
            href={null}
            onClick={[Function]}
          >
            <span>
              file.xml
            </span>
            <button
              className="delete is-small"
              disabled={true}
              onClick={[Function]}
            />
          </a>
        </li>
      </ul>
    </div>
    <input
      className="input files-input"
      defaultValue="/this/is/the/full/filename.xml"
      disabled={true}
      placeholder="Filename"
      type="text"
    />
  </div>
</div>
`;
